src/desktop/apps/article/components/App.js

```diff
@@ -6,9 +6,12 @@ import { Article } from 'reaction/Components/Publishing'
 import { ContextProvider } from 'reaction/Components/Artsy'
 import { InfiniteScrollNewsArticle } from './InfiniteScrollNewsArticle.tsx'
 import { EditButton } from 'desktop/apps/article/components/EditButton'
-import ArticleLayout from './layouts/Article'
+import _ArticleLayout from './layouts/Article'
 import { data as sd } from 'sharify'
 
+// FIXME: Rewire
+let ArticleLayout = _ArticleLayout
```

Comment:
This can go away now, right?

Reply:
I mentioned it in the description above, but I believe since we haven't added in the latest version of `babel-rewire-plugin` we still need this to be able to rewire. Fwiw I did try to do this at one point but gave up after timeboxing an issue where one of the `tsx` files wasn't playing nice.  I'll give it another shot after we wrap up this sprint's work.

Reply:
Oh I see :) I mis-read and was thinking that those were the items that were added to this PR